snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: xinyi-pc-classic
-adopt-info: pc
+version: '22-0.4'
```

Comment:
Imho version should be set to something meaningful, like shim-signed + grub-efi-amd64-signed versions, using a version part. Given this is now following the archive, and doesn't have Ubuntu Core meaning.

Reply:
I think having a version as what you say would be too long, so I have changed it to something using 22.04 so it is clearer this is for classic. But it is a good idea that we know where things came from, so I've now primed `usr/share/doc/` for shim/grub.